Tiltfile

```
@@ -360,20 +360,26 @@ def deploy_worker_templates(template, substitutions):
         labels = ["flavors"],
     )
 
+def base64_local_args():
+    # Some versions of base64 require the "-i/--input" flag to read from stdin.
+    if local("base64 --help | grep '\\-i '", quiet = True, echo_off = True):
+        return "-i "
+    return ""
+
 def base64_encode(to_encode):
-    encode_blob = local("echo '{}' | tr -d '\n' | base64 - | tr -d '\n'".format(to_encode), quiet = True, echo_off = True)
+    encode_blob = local("echo '{}' | tr -d '\n' | base64 {}- | tr -d '\n'".format(to_encode, base64_local_args()), quiet = True, echo_off = True)
```

Would it be reasonable to omit this flag altogether and let the stdin default apply? I played around with that on Ubuntu and pre-Ventura macOS and it seems to work as expected.
```suggestion
    encode_blob = local("echo '{}' | tr -d '\n' | base64 | tr -d '\n'".format(to_encode), quiet = True, echo_off = True)
```

---

Yes, simpler == better if that works. Let do some more local testing and I'll update this.

---

Yep, much simpler and seems to work portably (between Ubuntu 22.04 and macOS 13 at least). 

Thanks @nojnhuh! Could you try this on macOS 12 or earlier?

---

Working for me!